coresimd/aarch64/neon.rs

```diff
@@ -380,6 +401,77 @@ pub unsafe fn vminvq_f64(a: float64x2_t) -> f64 {
     vminvq_f64_(a)
 }
 
+/// SHA1 hash update accelerator, choose.
+#[inline]
+#[target_feature(enable = "neon")]
```

Comment:
have you tried enabling "neon,crypto" here ?

Reply:
they probably just require the crypto feature

Reply:
Yeah, (I did `#[target_feature(enable = "neon", enable = "crypto")]`).  No luck.